scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -657,7 +655,6 @@ private static BuildTargetResult CompileCliSdk(BuildTargetContext c, DotNetCli d
                 {
                     // Yes, it is .exe even on Linux. This is the managed exe we're working with
                     File.Copy(Path.Combine(binaryToCorehostifyOutDir, $"{binaryToCorehostify}.exe"), Path.Combine(outputDir, $"{binaryToCorehostify}.dll"));
-                    File.Delete(Path.Combine(binaryToCorehostifyOutDir, $"{binaryToCorehostify}.exe"));
```

Comment:
?? I think we should continue deleting the `csc.exe` file, since it isn't being used - we don't want it in our SDK folder.